### ISSUE TITLE
Login: Part I

### ISSUE THREAD

**Issue by eleddy** [plone/Products.CMFPlone#105]
I had huge plans to rework login completely but this might fall out of scope of what we can actually do in a small amount of time. Instead, lets integrate gils branch and then make plans after that. The main thing we need to add in this release are TEST CASES (non doc test test cases). Be smart about them and don't do features that are likely to deprecate immediately. You might want to just remove things as well, for the sake of this being a major release and having the opportunity to do so.

@esteele: How angry do you get thinking about having login as a separate package? Does it makes sense?

@gforcada 's work on the migration is at: https://github.com/gforcada/login_forms

Once that is merged in, remove everything at https://github.com/plone/Products.CMFPlone/tree/master/Products/CMFPlone/skins/plone_login

Other relevant files:

 * Products/CMFPlone/skins/plone_ecmascript/login.js
 * Products/CMFPlone/tests/LoginAndLogout.txt
 * Products/CMFPlone/tests/testSSOLogin.py
 * Products/CMFPlone/tests/emaillogin.txt
 * Products/CMFPlone/tests/testEmailLogin.py
 * Products/CMFPlone/tests/testLoggedIn.py

**Comment by tomgross** [comment]
What is the state of plone.login?

**Comment by vangheem** [comment]
@tomgross https://github.com/plone/plone.login/issues/38

**Comment by vangheem** [comment]
Closing this. A plone.login plip needs to be written and an issue created for it.
